subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/LintSlackAlert.kt

```diff
@@ -0,0 +1,65 @@
+package com.avito.android.lint
+
+import com.avito.slack.SlackClient
+import com.avito.slack.model.SlackChannel
+import com.avito.utils.logging.CILogger
+
+interface LintSlackAlert {
```

Comment:
Alert is a usage scenario on the client-side.
Maybe it's better to use the term "notify" in the code.